server/src/main/scala/com/cloudera/livy/utils/SparkYarnApp.scala

```diff
@@ -78,7 +78,8 @@ class SparkYarnApp private[utils] (
   private var yarnDiagnostics: IndexedSeq[String] = IndexedSeq.empty[String]
 
   override def log(): IndexedSeq[String] =
-    process.map(_.inputLines).getOrElse(ArrayBuffer.empty[String]) ++ yarnDiagnostics
+    process.map(_.inputLines).getOrElse(ArrayBuffer.empty[String]) ++ yarnDiagnostics ++
+      process.map(_.errorLines).getOrElse(ArrayBuffer.empty[String])
```

Comment:
for some reason driver process output is coming from errorStream. So added error lines to log.